src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  let query = '';
+  const [visibleMovies, setVisibleMovies] = useState([...moviesFromServer]);
+
+  const filteredMovies = (movieList: Movie[], searchQuery: string) => {
+    return [...movieList].filter(movie => (
+      movie.description.toLowerCase().includes(searchQuery.toLowerCase())
+      || movie.title.toLowerCase().includes(searchQuery.toLowerCase())));
```

Comment:
create a separate function for it, since you are using same logic twice

Reply:
Hi, Nazar! Thanks for review. Yes, you're right. I've done.